### ISSUE TITLE
`mvn package` fails with missing symbols

### ISSUE THREAD

**Issue by nota-ja** [Macchinetta/atrs-cloud#1]
## Description

Running `mvn package` directly under this repository fails with `error: cannot find symbol`s:
```console
$ mvn package
....
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/FlightRepository.java:[41,40] error: cannot find symbol
[ERROR]   symbol:   class VacantSeatSearchCriteriaDto
  location: interface FlightRepository
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/FlightRepository.java:[50,31] error: cannot find symbol
[ERROR]   symbol:   class VacantSeatSearchCriteriaDto
  location: interface FlightRepository
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/masterdata/helper/BoardingClassHelper.java:[33,53] error: cannot find symbol
[ERROR]   symbol:   class BoardingClassRepository
  location: package jp.co.ntt.atrs.domain.repository.boardingclass
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/masterdata/helper/BoardingClassHelper.java:[47,4] error: cannot find symbol
[ERROR]   symbol:   class BoardingClassRepository
  location: class BoardingClassHelper
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[47,12] error: cannot find symbol
[ERROR]   symbol:   class RoutingDataSourceLookupKeyHolder
  location: class RoutingDataSource
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[54,29] error: cannot find symbol
[ERROR]   symbol:   class RoutingDataSourceBuilder
  location: class RoutingDataSource
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[55,12] error: cannot find symbol
[ERROR]   symbol:   class RoutingDataSourceLookupKeyHolder
  location: class RoutingDataSource
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/b1/TicketSearchServiceImpl.java:[56,46] error: cannot find symbol
[ERROR]   symbol:   class VacantSeatSearchCriteriaDto
  location: package jp.co.ntt.atrs.domain.repository.flight
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/c1/MemberRegisterServiceImpl.java:[34,52] error: cannot find symbol
[ERROR]   symbol:   class AccountShardKeyRepository
  location: package jp.co.ntt.atrs.domain.common.shard.repository
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/c1/MemberRegisterServiceImpl.java:[81,4] error: cannot find symbol
[INFO] 10 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for AIRLINE TICKET RESERVATION SYSTEM Parent POM 1.1.1.RELEASE:
[INFO]
[INFO] AIRLINE TICKET RESERVATION SYSTEM Parent POM ....... SUCCESS [  0.088 s]
[INFO] atrs-common ........................................ SUCCESS [  1.545 s]
[INFO] atrs-config ........................................ SUCCESS [  0.458 s]
[INFO] atrs-model ......................................... SUCCESS [  0.158 s]
[INFO] AIRLINE TICKET RESERVATION SYSTEM Front Server ..... SUCCESS [  0.002 s]
[INFO] atrs-env ........................................... SUCCESS [  0.047 s]
[INFO] atrs-domain ........................................ FAILURE [  0.741 s]
[INFO] atrs-initdb ........................................ SKIPPED
[INFO] atrs-web ........................................... SKIPPED
[INFO] AIRLINE TICKET RESERVATION SYSTEM Back Server ...... SKIPPED
[INFO] atrs-back-env ...................................... SKIPPED
[INFO] atrs-back-domain ................................... SKIPPED
[INFO] atrs-back-web ...................................... SKIPPED
[INFO] atrs-integration-test .............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.589 s
[INFO] Finished at: 2020-05-12T13:15:13+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.5:compile (default-compile) on project atrs-domain: Compilation failure: Compilation failure:
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/FlightRepository.java:[41,40] error: cannot find symbol
[ERROR]   symbol:   class VacantSeatSearchCriteriaDto
[ERROR]   location: interface FlightRepository
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/FlightRepository.java:[50,31] error: cannot find symbol
[ERROR]   symbol:   class VacantSeatSearchCriteriaDto
[ERROR]   location: interface FlightRepository
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/masterdata/helper/BoardingClassHelper.java:[33,53] error: cannot find symbol
[ERROR]   symbol:   class BoardingClassRepository
[ERROR]   location: package jp.co.ntt.atrs.domain.repository.boardingclass
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/masterdata/helper/BoardingClassHelper.java:[47,4] error: cannot find symbol
[ERROR]   symbol:   class BoardingClassRepository
[ERROR]   location: class BoardingClassHelper
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[47,12] error: cannot find symbol
[ERROR]   symbol:   class RoutingDataSourceLookupKeyHolder
[ERROR]   location: class RoutingDataSource
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[54,29] error: cannot find symbol
[ERROR]   symbol:   class RoutingDataSourceBuilder
[ERROR]   location: class RoutingDataSource
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[55,12] error: cannot find symbol
[ERROR]   symbol:   class RoutingDataSourceLookupKeyHolder
[ERROR]   location: class RoutingDataSource
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/b1/TicketSearchServiceImpl.java:[56,46] error: cannot find symbol
[ERROR]   symbol:   class VacantSeatSearchCriteriaDto
[ERROR]   location: package jp.co.ntt.atrs.domain.repository.flight
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/c1/MemberRegisterServiceImpl.java:[34,52] error: cannot find symbol
[ERROR]   symbol:   class AccountShardKeyRepository
[ERROR]   location: package jp.co.ntt.atrs.domain.common.shard.repository
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/c1/MemberRegisterServiceImpl.java:[81,4] error: cannot find symbol
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :atrs-domain
```

リポジトリー直下で `mvn package` を実行すると， `エラー: シンボルを見つけられません` が発生して失敗する:
```console
$ mvn package
....
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/FlightRepository.java:[41,40] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス VacantSeatSearchCriteriaDto
  場所: インタフェース FlightRepository
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/FlightRepository.java:[50,31] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス VacantSeatSearchCriteriaDto
  場所: インタフェース FlightRepository
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/masterdata/helper/BoardingClassHelper.java:[33,53] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス BoardingClassRepository
  場所: パッケージ jp.co.ntt.atrs.domain.repository.boardingclass
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/masterdata/helper/BoardingClassHelper.java:[47,4] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス BoardingClassRepository
  場所: クラス BoardingClassHelper
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[47,12] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス RoutingDataSourceLookupKeyHolder
  場所: クラス RoutingDataSource
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[54,29] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス RoutingDataSourceBuilder
  場所: クラス RoutingDataSource
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[55,12] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス RoutingDataSourceLookupKeyHolder
  場所: クラス RoutingDataSource
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/b1/TicketSearchServiceImpl.java:[56,46] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス VacantSeatSearchCriteriaDto
  場所: パッケージ jp.co.ntt.atrs.domain.repository.flight
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/c1/MemberRegisterServiceImpl.java:[34,52] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス AccountShardKeyRepository
  場所: パッケージ jp.co.ntt.atrs.domain.common.shard.repository
/Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/c1/MemberRegisterServiceImpl.java:[81,4] エラー: シンボルを見つけられません
[INFO] 10 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for AIRLINE TICKET RESERVATION SYSTEM Parent POM 1.1.1.RELEASE:
[INFO]
[INFO] AIRLINE TICKET RESERVATION SYSTEM Parent POM ....... SUCCESS [  0.079 s]
[INFO] atrs-common ........................................ SUCCESS [  1.481 s]
[INFO] atrs-config ........................................ SUCCESS [  0.442 s]
[INFO] atrs-model ......................................... SUCCESS [  0.157 s]
[INFO] AIRLINE TICKET RESERVATION SYSTEM Front Server ..... SUCCESS [  0.002 s]
[INFO] atrs-env ........................................... SUCCESS [  0.046 s]
[INFO] atrs-domain ........................................ FAILURE [  0.700 s]
[INFO] atrs-initdb ........................................ SKIPPED
[INFO] atrs-web ........................................... SKIPPED
[INFO] AIRLINE TICKET RESERVATION SYSTEM Back Server ...... SKIPPED
[INFO] atrs-back-env ...................................... SKIPPED
[INFO] atrs-back-domain ................................... SKIPPED
[INFO] atrs-back-web ...................................... SKIPPED
[INFO] atrs-integration-test .............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.407 s
[INFO] Finished at: 2020-05-12T13:34:24+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.5:compile (default-compile) on project atrs-domain: Compilation failure: Compilation failure:
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/FlightRepository.java:[41,40] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス VacantSeatSearchCriteriaDto
[ERROR]   場所: インタフェース FlightRepository
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/FlightRepository.java:[50,31] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス VacantSeatSearchCriteriaDto
[ERROR]   場所: インタフェース FlightRepository
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/masterdata/helper/BoardingClassHelper.java:[33,53] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス BoardingClassRepository
[ERROR]   場所: パッケージ jp.co.ntt.atrs.domain.repository.boardingclass
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/masterdata/helper/BoardingClassHelper.java:[47,4] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス BoardingClassRepository
[ERROR]   場所: クラス BoardingClassHelper
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[47,12] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス RoutingDataSourceLookupKeyHolder
[ERROR]   場所: クラス RoutingDataSource
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[54,29] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス RoutingDataSourceBuilder
[ERROR]   場所: クラス RoutingDataSource
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSource.java:[55,12] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス RoutingDataSourceLookupKeyHolder
[ERROR]   場所: クラス RoutingDataSource
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/b1/TicketSearchServiceImpl.java:[56,46] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス VacantSeatSearchCriteriaDto
[ERROR]   場所: パッケージ jp.co.ntt.atrs.domain.repository.flight
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/c1/MemberRegisterServiceImpl.java:[34,52] エラー: シンボルを見つけられません
[ERROR]   シンボル:   クラス AccountShardKeyRepository
[ERROR]   場所: パッケージ jp.co.ntt.atrs.domain.common.shard.repository
[ERROR] /Users/nota/repos/atrs-cloud/atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/service/c1/MemberRegisterServiceImpl.java:[81,4] エラー: シンボルを見つけられません
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :atrs-domain
```

## Possible Solution(s)

TBD.

## Affected Version(s)

1.1.1.RELEASE

**Comment by nota-ja** [comment]
The cause of this issue was that there were missing files (see below) with [1.1.1.RELEASE](https://github.com/Macchinetta/atrs-cloud/tree/1.1.1.RELEASE).

These files are included in 1.1.0.RELEASE, but not in 1.1.1.RELEASE. So this issue only affects on 1.1.1.RELEASE.

And the files were added with commit 49392ceef98206b42282fa3a8b97a7b243807f62, so the issue should have been solved.

本issueの原因は，[1.1.1.RELEASE](https://github.com/Macchinetta/atrs-cloud/tree/1.1.1.RELEASE) において不足しているファイル（下記参照）が存在したことによるものだった。

これらのファイルは 1.1.0.RELEASE には含まれているが， 1.1.1.RELEASE には含まれていない。したがって本issueは 1.1.1.RELEASE にのみ影響するものである。

そしてこれらのファイルはコミット 49392ceef98206b42282fa3a8b97a7b243807f62 で追加されたので，問題は解決したものと考えられる。

---

* atrs-back/atrs-back-domain/src/main/java/jp/co/ntt/atrs/domain/common/exception/DuplicateReceivingException.java
* atrs-back/atrs-back-domain/src/main/java/jp/co/ntt/atrs/domain/service/b2/ReservationInspectionServiceImpl.java
* atrs-back/atrs-back-domain/src/main/resources/jp/co/ntt/atrs/domain/repository/messaging/MessageIdRepository.xml
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/security/AtrsAuthenticationFailureBadCredentialsListener.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/security/AtrsAuthenticationSuccessListener.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/DataSourceKeyResolver.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSourceBuilder.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/RoutingDataSourceLookupKeyHolder.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/model/CommonDatabaseProperties.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/model/DatabaseProperties.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/pool/AmazonRdsReadReplicaTomcatDataSourceFactory.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/datasource/pool/TomcatDataSourceFactory.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/interceptor/AccountShardInterceptor.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/common/shard/repository/AccountShardKeyRepository.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/boardingclass/BoardingClassRepository.java
* atrs-front/atrs-domain/src/main/java/jp/co/ntt/atrs/domain/repository/flight/VacantSeatSearchCriteriaDto.java
* atrs-front/atrs-domain/src/main/resources/jp/co/ntt/atrs/domain/repository/boardingclass/BoardingClassRepository.xml
* atrs-front/atrs-domain/src/main/resources/jp/co/ntt/atrs/domain/repository/reservation/ReservationRepository.xml
* atrs-front/atrs-web/src/main/java/jp/co/ntt/atrs/app/common/security/AtrsUsernamePasswordAuthenticationFilter.java

**Comment by nota-ja** [comment]
cf. https://github.com/Macchinetta/atrs-cloud/runs/665836364
